src/coreclr/build.cmd

```diff
@@ -598,23 +613,26 @@ if %__BuildCoreLib% EQU 1 (
     )
 
     set __BuildLogRootName=System.Private.CoreLib
-    set __BuildLog="%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-    set __BuildWrn="%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
-    set __BuildErr="%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
-    set __MsbuildLog=/flp:Verbosity=normal;LogFile=!__BuildLog!
-    set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
-    set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!
-    set __Logging=!__MsbuildLog! !__MsbuildWrn! !__MsbuildErr!
+    set "__BuildLog=%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
+    set "__BuildWrn=%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
+    set "__BuildErr=%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
+    set "__Binlog=%__LogsDir%\!__BuildLogRootName!_%__BuildOS%__%__BuildArch%__%__BuildType%.binlog"
+    set "__MsbuildLog=/flp:Verbosity=normal;LogFile=!__BuildLog!"
+    set "__MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!"
+    set "__MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!"
+    set "__MsbuildBinLog=/bl:!__Binlog!"
+    set "__Logging=!__MsbuildLog! !__MsbuildWrn! !__MsbuildErr! !__MsbuildBinLog!"
 
     powershell -NoProfile -ExecutionPolicy ByPass -NoLogo -File "%__RepoRootDir%\eng\common\msbuild.ps1" /clp:nosummary %__ArcadeScriptArgs%^
         %__ProjectDir%\src\build.proj /t:Restore^
         /nodeReuse:false /p:PortableBuild=true /maxcpucount /p:IncludeRestoreOnlyProjects=true^
-        !__Logging! %__CommonMSBuildArgs% !__ExtraBuildArgs! %__UnprocessedBuildArgs%
+        !__Logging! /bl: %__CommonMSBuildArgs% !__ExtraBuildArgs! %__UnprocessedBuildArgs%
```

Comment:
@jkoritzinsky Is there a reason restore is separate from build? The only reason I saw was "optdata.csproj" getting referenced with `IncludeRestoreOnlyProjects` but this was already restored manually previously and only brings in the PGO data which shouldn't be needed for SPC. Is there also a reason why restore is different from getting the pgo path steps instead of passing restore? This would be two less invocations of msbuild.

Reply:
We should be able to switch the separate restore and build to instead be `/restore /t:Build` so both are done in one invocation.

Reply:
I'll do this in a follow-up PR then.